react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -317,6 +324,16 @@ public void requestMediaEditor(MediaUploadCallback mediaUploadCallback, String m
                 mPendingMediaUploadCallback = mediaUploadCallback;
                 mOnMediaEditorListener.onMediaEditorClicked(mediaUrl);
             }
+
+            @Override
+            public void logUserEvent(GutenbergUserEvent event, ReadableMap eventProperties) {
+                switch (event) {
+                    case PAGE_TEMPLATE_APPLIED:
+                        mOnLogGutenbergUserEventListener.onGutenbergDidLogSessionTemplateEvent(eventProperties.getString(
```

Comment:
I'm not familiar with conventions on WPAndroid or Kotlin, but I wonder if it'd be less verbose if we had just one callback method and let the app switch through the different cases of the enum?

Reply:
I agree. Initially, I tried to multiplex the entire "interface chain" to reduce the total usage of forwarding methods, but couldn't use `ReadableMap` from the parent app scope. I was able to partially address this with the `toHashmap` method in this commit: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1945/commits/a7b991d6ae7567918917ec24c54dd526f2f78958.